src/components/MiniMap.jsx

```diff
@@ -0,0 +1,61 @@
+import T from "prop-types";
+import MapGl, { Layer, Source } from "react-map-gl/maplibre";
+import "maplibre-gl/dist/maplibre-gl.css";
+import bbox from "@turf/bbox";
+import { AttributionControl } from "react-map-gl";
```

Comment:
AttributionControl needs to be imported from "react-map-gl/maplibre"

```suggestion
import MapGl, { Layer, Source, AttributionControl } from "react-map-gl/maplibre";
import "maplibre-gl/dist/maplibre-gl.css";
import bbox from "@turf/bbox";
```